prac2-pages/Simranpreet Kaur/login2.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:flutter/material.dart';
+import 'login_page.dart';
+
+class Login2 extends StatefulWidget{
+  const Login2({Key? key}) : super(key:key);
+
+  @override
+  State<Login2> createState() => _Login2State();
+
+}
+class _Login2State extends State<Login2>
+{ var val2 = false;
+  String? str1 ;
+  var check = false;
+  @override
+  Widget build(BuildContext context){
+    return Scaffold(
+      backgroundColor: Colors.white,
+      body:SafeArea(
+        child :Center(
+          child:Column(
+            children:  [
+              SizedBox(height: 55,),
+              //Login Verification
+              Text('Login Verification',
+              style : TextStyle(
+                fontWeight: FontWeight.bold,
+                fontSize: 30,
+                color: Color(0xFF06061C),
+              )),
+              SizedBox(height: 25,),
+              Center(
+                child: Padding(
+                  padding: EdgeInsets.fromLTRB(25,10,25,10),
+                  child: Text('To continue logging in we need you to complete one of the following:',
+                  textAlign: TextAlign.center,
+                  style : TextStyle(
+                    fontSize: 20,
+                  )),
+                ),
+              ),
+                SizedBox(height:25 ,),
+
+              RadioListTile(
+                title: Text("Verification code via email"),
+                activeColor: Color(0xFF06061C),
+                value: "email",
+                groupValue: str1,
+                onChanged: ((value) {
+                  setState(() {
+                    str1 = value.toString();
+                  });
+                }),
+              ),
+              RadioListTile(
+                title: Text("Verification code via SMS"),
+                activeColor: Color(0xFF06061C),
+                value: "SMS",
+                groupValue: str1,
+                onChanged: ((value) {
+                  setState(() {
+                    str1 = value.toString();
+                  });
+                }),
+              ),
+
+
+
+          SizedBox(height: 10,),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(60,0,25,10),
+            child: Text(
+                'We\'ll send a code to your mobile phone that can be entered to verify your identity.',
```

Comment:
Suggestion: :thought_balloon: 
- Make it so that it shows different texts depending on the user selection of verification by email or SMS
- You may need to use boolean operators to achieve this effect

Reply:
Implemented gracefully! :+1: